### DISCUSSION TITLE
using wildcard(*) in serverDependenciesToBundle

### DISCUSSION THREAD

**Discussion by CodePsy-2001** [remix-run/remix#3855] [Proposals]
Hello!

I'm using sendbird with my remix app.
sendbird UI kit does not supporte named export, so serverDependenciesToBunlde array is to be....

```javascript
serverDependenciesToBundle: [
    'sendbird',
    '@sendbird/uikit-react/App',
    '@sendbird/uikit-react/ChannelList',
    '@sendbird/uikit-react/SendbirdProvider',
    // ....
],
```

I want to abbreviate this! like this....

```javascript
serverDependenciesToBundle: [
    'sendbird',
    '@sendbird/uikit-react/*',
    // ....
],
```

If you don't like this offer, please let me know why. I like to learn in the community!
Thank you, and sorry for my poor English.

**Comment by Xiphe** [comment]
I was under the assumption that

```
serverDependenciesToBundle: [
    'sendbird',
    '@sendbird/uikit-react',
    // ....
],
```

should work in this case? Mind giving it a try?

**Reply by CodePsy-2001** [reply]
It just not work...

**Reply by arsen91** [reply]
You ever got to a solution? Have you raised a ticket?

**Reply by arsen91** [reply]
This thread helped me: https://github.com/remix-run/remix/discussions/2594
